### PR TITLE
Iteration 1

### PR DESCRIPTION
**Pull request by Ashkanthegreat**
Created Game, Game Teams, and teams files and tests.  Completed Git Ignore and Runner files.

### CODE REVIEW

File: test/team_test.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    assert_equal "/api/v1/teams/1", @team.link
+  end
+
+  def test_csv_access
+    assert_equal 53, @csv_team.team_id
+    assert_equal 28, @csv_team.franchise_id
+    assert_equal "Columbus Crew SC", @csv_team.team_name
+    assert_equal "CCS", @csv_team.abbreviation
+    assert_equal "Mapfre Stadium", @csv_team.stadium
+    assert_equal "/api/v1/teams/53", @csv_team.link
```
Comment (jpc20) [inline test/team_test.rb:37]: Looks good!

File: test/fixtures/games_fixture.csv [hunk 2]
```diff
@@ -0,0 +1,12 @@
+game_id,season,type,date_time,away_team_id,home_team_id,away_goals,home_goals,venue,venue_link
```
Comment (arpariseau) [inline test/fixtures/games_fixture.csv:1]: We wanted to pull out a selection that had both postseason and regular season games for the fixtures.

File: lib/game_team.rb [hunk 3]
```diff
@@ -1,6 +1,17 @@
 class GameTeam
 
+  attr_reader :game_id,
```
Comment (Ashkanthegreat) [inline lib/game_team.rb:16]: We should change attr readers in other files to match this.

File: test/fixtures/games_teams_fixture.csv [hunk 4]
```diff
@@ -0,0 +1,23 @@
+game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways
```
Comment (arpariseau) [inline test/fixtures/games_teams_fixture.csv:1]: These match with the games we pulled out in our other fixture files

**Review by jpc20 — APPROVED** [review]
Nice work guys. 
Maybe we can refactor the csv reading to a parent class later on.

File: test/game_team_test.rb [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        game_id: "2012030221",
+        team_id: "3",
+        hoa: "away",
+        result: "LOSS",
+        settled_in: "OT",
+        head_coach: "John Tortorella",
+        goals: '2',
+        shots: '8',
+        tackles: '44',
+        pim: '8',
```
Comment (arpariseau) [inline test/game_team_test.rb:17]: I didn't know there were penalties in soccer!

File: test/game_team_test.rb [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+        powerplayopportunities: '3',
+        powerplaygoals: '0',
+        faceoffwinpercentage: '44.8',
+        giveaways: '17',
+        takeaways: '7'
+        })
+
+    GameTeam.from_csv("./test/fixtures/games_teams_fixture.csv")
```
Comment (EHale64) [inline test/game_team_test.rb:25]: Great name for the last game!

File: test/game_team_test.rb [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+  def test_it_has_readable_attributes
+    assert_equal 2012030221, @base_game_team.game_id
+    assert_equal 3, @base_game_team.team_id
+    assert_equal "away", @base_game_team.hoa
+    assert_equal "LOSS", @base_game_team.result
+    assert_equal "OT", @base_game_team.settled_in
+    assert_equal "John Tortorella", @base_game_team.head_coach
+    assert_equal 2, @base_game_team.goals
+    assert_equal 8, @base_game_team.shots
+    assert_equal 44, @base_game_team.tackles
```
Comment (EHale64) [inline test/game_team_test.rb:42]: Tackles in Soccor?

**Review by EHale64 — APPROVED** [review]
Looks Good

File: lib/game_team.rb [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+              :team_id,
+              :hoa,
+              :result,
+              :settled_in,
+              :head_coach,
+              :goals,
+              :shots,
+              :tackles,
+              :pim,
```
Comment (arpariseau) [inline lib/game_team.rb:25]: I didn't know there were penalties in soccer!

File: test/game_team_test.rb (hunk 7 above)
Comment (arpariseau) [reply test/game_team_test.rb:42]: It's when you take the ball away!

File: lib/game_team.rb [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+              :powerplayopportunities,
```
Comment (arpariseau) [inline lib/game_team.rb:26]: or power plays...

File: lib/game_team.rb [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+              :powerplaygoals,
+              :faceoffwinpercentage,
```
Comment (arpariseau) [inline lib/game_team.rb:28]: or face-offs!

**Pull request merged**